web/cobrands/shropshire/assets.js

```diff
@@ -52,16 +47,20 @@ fixmystreet.assets.add(defaults, {
      }),
     always_visible: true,
     non_interactive: true,
+    usrn: {
+        attribute: 'USRN',
+        field: 'site_code'
+    },
     road: true,
```

Comment:
`usrn:` looks right, yep, but you still want to drop this line (although all it's doing is looking for hooks to call that aren't there, or setting body_send which doesn't matter here, I think, but best to be clear you're not using the functionality).